labonneboite/common/search.py

```diff
@@ -235,21 +236,27 @@ def get_companies(self, add_suggestions=False):
         self.company_count = self._get_company_count(self.rome, self.distance)
         logger.debug("set company_count to %s from fetch_companies", self.company_count)
 
-        if self.from_number < 1:
-            self.from_number = 1
-            self.to_number = 10
-        if (self.from_number - 1) % 10:
-            self.from_number = 1
-            self.to_number = 10
+        max_page_size = pagination.OFFICES_MAXIMUM_PAGE_SIZE
+        current_page_size = self.to_number - self.from_number + 1
+        
```

Comment:
Trailing whitespace :white_flag: 
Sur mon éditeur, j'ai un plug-in qui surligne ces espaces inutiles. Peux-être qu'il en existe un pour Sublime Text ?

Reply:
bien vu, et pylint ne le remarquait pas...

vu que pylint est censé être notre standard de qualité de code, soit on considère que ton éditeur est trop "picky" ;) soit il faut tuner la conf de notre `.pylintrc` pour le rendre aussi picky que ton éditeur sur ce point particulier

je vote pour le premier  ;)

d'ailleurs c'est quoi ton éditeur et/ou notre pylint est-il bien intégré dedans? @AlexandreCantin